sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/CheckDownloadTask.java

```diff
@@ -191,4 +184,22 @@ public void run() {
             });
         }
     }
+
+    /**
+     * Store details about downloaded release.
+     * After app update and restart, this info is used to report new download and to update group ID (if it's changed).
+     */
+    private void storeDownloadedReleaseDetails() {
+        if (mReleaseDetails == null) {
+            AppCenterLog.debug(LOG_TAG, "Downloaded release details are missing or broken, won't store.");
+            return;
+        }
+        String groupId = mReleaseDetails.getDistributionGroupId();
+        String releaseHash = mReleaseDetails.getReleaseHash();
+        int releaseId= mReleaseDetails.getId();
```

Comment:
please format file with Android Studio. It's recommended to use android studio commit UI to have the automated actions for that (cleanup, organize imports, reformat, code analysis).

Reply:
Fixed. Thank you for the advice.